package.json

```diff
@@ -20,7 +20,7 @@
     "lodash": "^4.17.21"
   },
   "devDependencies": {
-    "@balena/lf-to-abstract-sql": "^4.6.0",
+    "@balena/lf-to-abstract-sql": "5.0.0-build-expand-lfinfo-cd6429c29d11d986d13b2f3a1c7090310e3e4d03-1",
```

Comment:
This should point to a real version before unmarking as draft (to avoid accidentally merging whilst pointing at a prerelease version)